front/app/containers/IdeasNewPage/WithJSONForm/index.tsx

```diff
@@ -182,6 +188,52 @@ const IdeasNewPageWithJSONForm = ({ params }: WithRouterProps) => {
       config = getMethodConfig(project.attributes.participation_method);
     }
   }
+
+  if (isNilOrError(project) || isNilOrError(config)) {
+    return null;
+  }
+
+  const userCanEditProject =
+    !isNilOrError(authUser) &&
+    canModerateProject(project.id, { data: authUser });
+  const showEditSurveyButton =
+    userCanEditProject && config.postType === 'nativeSurvey';
+
+  const linkToSurveyBuilder = phaseId
+    ? `/admin/projects/${project.id}/phases/${phaseId}/native-survey/edit`
+    : `/admin/projects/${project.id}/native-survey/edit`;
+
+  const TitleComponent = (
+    <Box width="100%" display="flex" flexDirection="column">
+      <Box width="100%" display="flex" justifyContent="center">
+        <Box
+          display="flex"
+          width="50%"
+          flexDirection="row"
+          justifyContent="space-around"
+          mb="14px"
+        >
+          <GoBackButton insideModal={false} projectId={project.id} />
+          <Box data-cy="e2e-edit-survey-link">
+            {showEditSurveyButton && (
+              <Button
+                icon="edit"
+                linkTo={linkToSurveyBuilder}
+                buttonStyle="secondary"
+                padding="5px 8px"
```

Comment:
Could we try a `buttonStyle="text"` instead here? I think it will match the GoBack button a bit better (and I think there is a really slight difference in the grey colour of the button with the page background)

Reply:
This was supposed to look like the `Edit Project` button but I think you make a good point with the go back link looking different. I wonder if it would make more sense to make it look like the go back link 🤔 

![image](https://user-images.githubusercontent.com/12659000/196868698-7631e85d-f1dc-4fd3-b699-3883c129c02b.png)